app/controllers/users_email_controller.rb

```diff
@@ -20,9 +20,7 @@ class UsersEmailController < ApplicationController
 
   before_action :require_login, only: [
     :confirm_old_email,
-    :show_confirm_old_email,
-    :confirm_new_email,
```

Comment:
Do we need to remove the following lines too? https://github.com/discourse/discourse/blob/3303b7f9d0a0e2a4cf310eadc9e4dc8fa9ed8572/app/controllers/users_email_controller.rb#L17-L18